models/user.js

```diff
@@ -145,15 +145,15 @@ exports.loginUser = function(username, password, callback = {success: () => {},
 
 						if (compareResult)
 						{
-							callback.success();
+							callback.success(user);
```

Comment:
where does this user object come from?
for exports.getUser, does the second parameter (the callback) need to be like
(user) => 
{
}

? or does it currently work as is?

Reply:
user varible does exist at line 137. I did run a quick test and it works but it's good to be sure.